### DISCUSSION TITLE
Spatial discretization

### DISCUSSION THREAD

**Discussion by alimostafavi24** [idaholab/moose#23966] [Q&A General] [answered]
Hello everyone,

I had a question regarding the spatial discretization in incompressible navier-stokes using the finite element method. The kind of the gird that is used there, is it staggered grid or collocated grid, and which part of the code I can see the information related to this?

Thanks,
Ali

**Comment by GiudGiud** [comment, marked as answer]
Hello

It s colocated, unless you use different orders for pressure and velocity, as is being worked on here
https://github.com/idaholab/moose/discussions/23929

Guillaume

**Comment by alimostafavi24** [comment]
Ok, so if I use 2nd order for the velocity components and 1st order for pressure, what will happen exactly?

Thanks,
Ali

**Reply by GiudGiud** [reply]
They will have different shapes over each element. I dont think it's called staggered but it also does not suffer from some of the ails of colocation.
